### ISSUE TITLE
Source data issue: "CCSDS 133.1-B-1 Pre-Corrigendum 1” should be "CCSDS 133.1-B-1-S”

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-data-ccsds#7]
https://public.ccsds.org/Publications/SilverBooks.aspx

This identifier actually points to https://public.ccsds.org/Pubs/133x1b1c1.pdf , which is just “CCSDS 133.1-B-1” (after retired it should be “CCSDS 133.1-B-1-S”).

If we look at the bigger picture, this document is actually “CCSDS 133.1-B-1-S” itself, not any special “pre-corrigendum” document.

Reasons:
1. On https://public.ccsds.org/Publications/SilverBooks.aspx, there is no "133.1-B-1-S” (issue 1), but "133.1-B-2-S” (issue 2) exists.
2. On https://public.ccsds.org/Publications/AllPubs.aspx, “133.1-B-3” (issue 3) exists.

Hence we know that this document is actually just "133.1-B-1-S”.

Proposed action: change identifier "CCSDS 133.1-B-1 Pre-Corrigendum 1” => "CCSDS 133.1-B-1-S”, change issue number from “1 Pre-Corrigendum 1” to just “1”.

<img width="859" alt="image" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/be4e0def-3ade-4f20-bd94-de0be6cbb0c4">

**Comment by andrew2net** [comment]
> Proposed action: change identifier "CCSDS 133.1-B-1 Pre-Corrigendum 1” => "CCSDS 133.1-B-1-S”, change issue number from “1 Pre-Corrigendum 1” to just “1”.

@ronaldtse but "CCSDS 133.1-B-1-S” already exists. And these documents aren't identical. For example, "CCSDS 133.1-B-1-S” has corrections that "CCSDS 133.1-B-1 Pre-Corrigendum 1” doesn't:

"CCSDS 133.1-B-1-S”
<img width="711" alt="image" src="https://github.com/relaton/relaton-data-ccsds/assets/5623890/9e857112-f83e-45d9-98af-8957d838f155">

"CCSDS 133.1-B-1 Pre-Corrigendum 1”
<img width="712" alt="image" src="https://github.com/relaton/relaton-data-ccsds/assets/5623890/c847d0a3-0712-448a-a19b-0ef2deecc949">

**Comment by ronaldtse** [comment]
@andrew2net great find. So we know that:

* CCSDS 133.1-B-1-S incorporates "Cor 1"
* CCSDS 133.1-B-1-S Pre-Corrigendum 1 does not incorporate "Cor 1"

The first item clearly was published in October 2006 not June 2006.

|CCSDS 133.1-B-1-S incorporating "Cor 1" | Pre-corrigendum 1 |
|--|--|
|<img width="688" alt="Screenshot 2023-08-15 at 11 41 02 AM" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/639210be-37e4-4659-98c3-749c4a22f960"> | <img width="678" alt="Screenshot 2023-08-15 at 11 41 12 AM" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/0dffc8bb-1a50-41b8-b08c-a7945ab84777"> |
|<img width="837" alt="Screenshot 2023-08-15 at 11 41 35 AM" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/900381ae-194b-4508-b4f5-4dc09ce5bb80"> | <img width="768" alt="Screenshot 2023-08-15 at 11 42 44 AM" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/ad199521-ed0f-4674-9ff0-6e2b2407019f"> |
| <img width="883" alt="Screenshot 2023-08-15 at 11 40 01 AM" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/1a1ed983-66d3-46b5-ae62-e7ef9d90d48e"> | <img width="845" alt="Screenshot 2023-08-15 at 11 39 38 AM" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/df2acc1c-3cfc-4cc6-83b2-1cdb2c1b448b"> |

An additional problem is that the footer of the "standard with cor" contains an identifier that is identical to the Tech Cor itself, "CCSDS 133.1-B-1 Cor. 1":
<img width="808" alt="Screenshot 2023-08-15 at 11 45 44 AM" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/b713d65d-f6f7-4ed7-ace6-87a032e62086">

We need to clarify this with CCSDS: the identifier of the standard that incorporated the cor, and the date/edition reference.

**Comment by ronaldtse** [comment]
My report to CCSDS:

I just realized that the problem regarding "CCSDS 133.1-B-1 Pre-Corrigendum” is slightly more complex.

There are two published "CCSDS 133.1-B-1-S” documents:

a. "CCSDS 133.1-B-1-S Pre-Corrigendum 1” that did not incorporate "CCSDS 133.1-B-1-S Cor. 1”, published June 2006
b. "CCSDS 133.1-B-1-S" that incorporates "CCSDS 133.1-B-1-S Cor. 1”, published October 2006

The timeline is this:
- "CCSDS 133.1-B-1-S” was published in June 2006
- "CCSDS 133.1-B-1-S Cor. 1” was published in October 2006
- "CCSDS 133.1-B-1-S” that incorporates "CCSDS 133.1-B-1-S Cor. 1” was republished in October 2006 as "CCSDS 133.1-B-1-S” (same old identifier), the original "CCSDS 133.1-B-1-S” was renamed into "CCSDS 133.1-B-1-S Pre-Corrigendum 1”.

The exact differences of content between a. and b. are illustrated in this ticket:
https://github.com/relaton/relaton-data-ccsds/issues/7#issuecomment-1678387573

This situation is interesting because the incorporation of "Cor. 1” did not increment the “Issue” number, which meant that the old publication identifier was re-used.

From what I can see of “Cor” documents in the CCSDS library, standards are rarely re-published to incorporate corrigenda. This seemed to be the only instance in the library?

I also don’t know whether "CCSDS 133.1-B-1-S Pre-Corrigendum 1” is the desired publication identifier because this is the only instance of a “pre-corrigendum” document in the CCSDS online library.

There are two questions that arose from this situation:

1. Reference stability. If I wanted to lock my citation to the originally published “113.1-B-1-S”, I have to change my citation to “113.1-B-1-S Pre-Corrigendum 1”. However, when I initially authored the citation (between June and October 2006), I would not know whether a corrigendum will be issued and cannot make the desired reference.

2. Publication footer. Between the documents a. and b., all pages use the footer reference identifier of "CCSDS 133.1-B-1-S”. However, in document b, the footer shows "CCSDS 133.1-B-1-S Cor. 1” mixed with "CCSDS 133.1-B-1-S”. This is an inconsistency in the footer of the October 2006 "CCSDS 133.1-B-1-S” document. However, "CCSDS 133.1-B-1-S Cor. 1” is the identifier of the Technical Corrigendum document, not the base document being corrected. So there is a confusion here.

I guess the ultimate question is: what is the correct way to distinguish "CCSDS 133.1-B-1-S Pre-Corrigendum 1” and "CCSDS 133.1-B-1-S”?

**Comment by andrew2net** [comment]
@ronaldtse it seems the "CCSDS 133.1-B-1 Pre-Corrigendum 1” is a first reduction. The "CCSDS 133.1-B-1-S Cor. 1” is a correction. And the "CCSDS 133.1-B-1-S” is a merging of the first reduction and the correction.

**Comment by ronaldtse** [comment]
@andrew2net yes, if "reduction" means "publication". We are waiting for CCSDS's reply.

**Comment by ronaldtse** [comment]
CCSDS reported this fixed. However, it seems that they now removed the Pre-Corrigendum document.

<img width="752" alt="image" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/d322a81a-1ff6-402e-aadb-7c2ed0366ce4">

Keeping this open until they have further responses on the Pre-Corrigendum.
